.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "7"
```

Comment:
+1, ive had builds fail because of version incompatibilities especially if you aren't using a yarn.lock or package-lock.json file